src/pages/[...paths].tsx

```diff
@@ -15,46 +16,92 @@ import React from 'react';
 import TextContent from 'modules/Common/components/TextContent';
 import ThumbGallery from 'modules/Common/components/ThumbGallery';
 import ThumbGalleryItem from 'modules/Common/components/ThumbGalleryItem';
+import { getCaseStudieSubtitle } from 'pages/case-studies';
+import { useRouter } from 'next/router';
 import useTranslation from 'next-translate/useTranslation';
 
 const FOLDER = 'pages';
 
-export default function PrivacyPolicyPage({ mdxContent }: WithMdxResult) {
+export default function StaticPage({ mdxContent }: WithMdxResult) {
   const { frontmatter = {} } = mdxContent || {};
   const { t } = useTranslation();
+  const router = useRouter();
+
+  const isCaseStudie = router.query.paths?.includes('case-studies');
+
+  let breadcrumbItems = new Array();
+  if (isCaseStudie) {
+    //Breadcrumb items
+    breadcrumbItems.push({
+      name: t('case-studies:breadcrumb.title'),
+      url: router.locale === 'fr' ? '/fr/case-studies' : '/case-studies',
+    });
+    breadcrumbItems.push({
+      name: frontmatter.title,
+    });
+  }
+  if (frontmatter.breadcrumb) {
+    breadcrumbItems = frontmatter.breadcrumb;
+  }
 
   return (
-    <Layout title={frontmatter?.title} desc={frontmatter?.description}>
+    <Layout
+      title={frontmatter?.seo?.title || frontmatter?.title}
+      desc={frontmatter?.seo?.description}
+    >
       {frontmatter['hero.title'] && (
         <Container layout="wide" dark={true} paddingY={false}>
-          <Container gridCols="12" gridGutters="11" flex={true} paddingX={false}>
+          <Container gridCols="12" gridGutters="11" flex={true} paddingX={false} dark={true}>
             <Hero title={frontmatter['hero.title']} subtitle={frontmatter['hero.subtitle']}></Hero>
           </Container>
         </Container>
       )}
+      {(frontmatter.breadcrumb || isCaseStudie) && (
+        <Container layout="wide" gray={true} paddingY={false}>
+          <Container gridCols="12" gridGutters="11" paddingYSmall={true}>
+            <Breadcrumb items={breadcrumbItems} className="mb__0"></Breadcrumb>
+          </Container>
+        </Container>
+      )}
 
-      <Container
-        gridCols={frontmatter['grid.cols'] ? frontmatter['grid.cols'] : 10}
-        gridGutters={frontmatter['grid.gutters'] ? frontmatter['grid.gutters'] : 9}
-      >
-        <TextContent>
-          {mdxContent && (
-            <MDXRemote
-              {...mdxContent}
-              components={{
-                Button,
-                LinkIcon,
-                Contributors,
-                Link,
-                ThumbGallery,
-                ThumbGalleryItem,
-              }}
-            />
-          )}
-        </TextContent>
-      </Container>
+      <Container paddingTop={false}>
+        <Container
+          gridCols={frontmatter['grid.cols'] || 10}
+          gridGutters={frontmatter['grid.gutters'] || 9}
+        >
+          <TextContent>
+            {frontmatter.title && isCaseStudie ? (
```

Comment:
I'm not sure to understand, can you explain otherwise?

Reply:
This layout is a global layout and should not contain specific `if`

The fact to have a margin can be interested for a global layout though.

So instead of testing if `caseStudy`, you could have a frontmatter entry called `titleMargin` so that contributor can select the margin he wants (even though I'm not sure it's a good idea)